### PR TITLE
Feature/home page with sass

### PR DESCRIPTION
**Pull request by MoneyManAnt06**
added some more html and hompage scss

### CODE REVIEW

File: src/scss/pages/homePage.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    padding: 120px 0 80px;
+  }
+}
+
+.AboutUs {
+  &-content {
+    max-width: 100%;
+    width: 100%;
+    display: flex;
+    background-color: #1c1c1c;
```
Comment (robSamuel) [inline src/scss/pages/homePage.scss]: Please add this hex into a variable.

File: src/scss/main.scss [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 @import "init";
 @import "about";
+@import "pages/homePage.scss";
```
Comment (robSamuel) [inline src/scss/main.scss]: Remove the extension of the file.

File: index.html [hunk 3]
```diff
@@ -1,83 +1,93 @@
-<!DOCTYPE html>
```
Comment (robSamuel) [inline index.html:1]: These two line should be included in your index.html

**Review by robSamuel — CHANGES_REQUESTED** [review]

File: src/scss/main.scss (hunk 2 above)
Comment (MoneyManAnt06) [reply src/scss/main.scss]: check

File: index.html (hunk 3 above)
Comment (MoneyManAnt06) [reply index.html:1]: check

Comment (MoneyManAnt06) [reply index.html:1]: check

**Review by robSamuel — APPROVED** [review]

**Pull request merged**